rundeckapp/grails-spa/packages/ui-trellis/src/library/components/plugins/tests/pluginPropVal.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { mount } from "@vue/test-utils";
+import PluginPropVal from "../pluginPropVal.vue";
+
+const createWrapper = (prop = {}, value: string | boolean = "") => {
+  return mount(PluginPropVal, {
+    props: {
+      prop: {
+        type: "String",
+        ...prop,
+      },
+      value,
+    },
+    global: {
+      mocks: {
+        $t: (msg: string) => msg,
+      },
```

Comment:
forgot to mention, there's no need to pass a global $t mock if the method is `(msg: string) => msg,`. Defined this method as the default mock in `jest.global-mocks.js`